chukwuemekachm/M-Tracker-React-Client
# #161280655 Completes feedback on user experience

#### What does this PR do?
- This PR implements feedback on the user experience
#### Description of Task to be completed?
- Modify userError handler method
- Update dashboard stlying of request lists
- Modify tests
- Add delete request component
- Test delete request component
#### How should this be manually tested?
- Clone the repo
- Start up command line
- Navigate to the project directory using `cd`
- Run `npm install`
- Run `npm start` or `npm run dev`
- Your default browser should be opened if you used `npm run dev`
- Login or signup(navigate to `/login` or `/signup`)
- You will be redirected to the dashboard observe the requests list
- Click on a pending, a resolved or a disapprove request to preview it
- Click the delete button on the top right corner of the page and observe
#### Production URL
- [heroku](https://m-tracker-react.herokuapp.com)
#### Any background context you want to provide?
- None
#### What are the relevant pivotal tracker stories?
- [#161280655](https://www.pivotaltracker.com/story/show/161280655)
#### Screenshots (if appropriate)

<img width="1440" alt="screenshot 2018-10-26 at 11 35 06 am" src="https://user-images.githubusercontent.com/33798252/47561489-4b8df280-d913-11e8-839d-c7373d8f6b13.png">

<img width="1440" alt="screenshot 2018-10-26 at 11 34 52 am" src="https://user-images.githubusercontent.com/33798252/47561488-4af55c00-d913-11e8-9de4-813ac7bcf9bb.png">


#### Questions:
- None
